tests/test_autodiff.py

```diff
@@ -46,6 +46,59 @@ def operator(kspace_loc, shape, backend, autograd):
     return nufft
 
 
+def proper_trajectory_torch(trajectory, normalize="pi"):
```

Comment:
@paquiteau this seems redundant code. Maybe we should discuss how to handle torch tensors as trajectories.